typescript/src/purchasesConfiguration.ts

```diff
@@ -34,16 +38,15 @@ export interface PurchasesConfiguration {
   userDefaultsSuiteName?: string;
   /**
    * iOS-only, will be ignored for Android.
-   * Set this to TRUE to enable StoreKit2.
-   * Default is FALSE.
+   * Default is STOREKIT_2.
```

Comment:
This is slightly strange considering we have a `DEFAULT` value in the enum... Should we add that the `DEFAULT` is  currently `STOREKIT_2` in the enum?